### PR TITLE
Updates for New Typo3 Publication Directory Structure

### PR DESCRIPTION
**Pull request by pmbittner**
The changes look a bit messy because they affected many parts of the Typo3 Export. I fixed some warnings (e.g., re-organized import declarations).

### CODE REVIEW

File: MibTeX/src/de/mibtex/FileUtils.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import java.io.File;
 
 public class FileUtils {
+    public final static String LINEBREAK = "\r\n";
+
```
Comment (tthuem) [inline MibTeX/src/de/mibtex/FileUtils.java]: Why not a system-independent call to System.lineSeparator?

File: MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java [hunk 2]
```diff
@@ -117,34 +146,78 @@ public void writeDocument() {
...
+
+            if (duplicates > 0) {
+                System.err.flush();
+                System.err.println("  There were unresolved duplicates that can cause problems when imported with TYPO3!");
+            }
+
+            writeToFileInUTF8(file, typo3);
+            exportedAFile = true;
+//            System.out.println();
+        } else {
```
Comment (tthuem) [inline MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java:201]: ?

File: MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java [hunk 3]
```diff
@@ -117,34 +146,78 @@ public void writeDocument() {
...
+        // Generate the typo3-conforming Bibtex source code.
+        final String typo3 = typo3Entries.stream()
+                .map(Typo3Entry::toString)
+                .reduce("", (a, b) -> a + "\n\n" + b);
+
+        System.out.println("=== EXPORTING " + t3dir.generatedFileName() + " ===");
+        if (!typo3.isBlank()) {
+//            System.out.println(typo3);
+//            System.out.println();
+
```
Comment (tthuem) [inline MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java]: ?

**Review by tthuem — APPROVED** [review]
Only minor stuff identified

File: MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java (hunk 2 above)
Comment (pmbittner) [reply MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java:201]: Removed commented line.

File: MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java (hunk 3 above)
Comment (pmbittner) [reply MibTeX/src/de/mibtex/export/ExportTypo3Bibtex.java]: Removed commented lines.

File: MibTeX/src/de/mibtex/FileUtils.java (hunk 1 above)
Comment (pmbittner) [reply MibTeX/src/de/mibtex/FileUtils.java]: changed to System.lineSeparator

**Pull request merged**